src/graph/graph.c

```
@@ -1440,17 +1439,15 @@ void Graph_memoryUsage
 	//--------------------------------------------------------------------------
 
 	D = Graph_GetAdjacencyMatrix(g, false);
-	M = Delta_Matrix_M(D);
 
-	info = GxB_Matrix_memoryUsage(&n, M);
+	info = Delta_Matrix_memoryUsage(&n, D);
 	ASSERT(info == GrB_SUCCESS);
```

Cppcheck (2.10-2)

[error] Null pointer dereference

(nullPointer)